### ISSUE TITLE
Using service-account with estimate regression since v1.01

### ISSUE THREAD

**Issue by ticoombs** [jay0lee/got-your-back#169]
When using --service-account it throws an error on an "estimate" command. 
`gyb --service-account --email my-email@domain.com --action estimate`

### Full steps to reproduce the issue:
1. Complete service-account flow
2. `git checkout v1.01`
3. `ln -s your-downloaded-service-account-key.json oauth2service.json`
4. `gyb --service-account --email my-email@domain.com --action estimate`

### Expected outcome (what are you trying to do?):
- Use `--service-account` with "--action estimate" argument to allow me to estimate the size of the email download . 

### Actual outcome (what errors or bad behavior do you see instead?):

```
Using backup folder GYB-GMail-Backup-my-email@domain.com
Traceback (most recent call last):
Traceback (most recent call last):
  File "gyb.py", line 1830, in <module>
    main(sys.argv[1:])
  File "gyb.py", line 1784, in main
    maxItems=500, fields='nextPageToken,messages/id')
  File "gyb.py", line 531, in callGAPIpages
    pageToken=pageToken, **kwargs)
  File "gyb.py", line 484, in callGAPI
    method = getattr(service, function)(**parameters)
  File "/home/timc/Documents/repos/got-your-back/googleapiclient/discovery.py", line 718, in method
    raise TypeError('Got an unexpected keyword argument "%s"' % name)
TypeError: Got an unexpected keyword argument "maxItems"
```

**Comment by pklapperich** [comment]
I've encountered this same bug without `--service-account`. It seems estimate is just broken in 1.01. Version 1.0 is working fine for me.

**Comment by pklapperich** [comment]
**WorkAround**
change line 1537 of gyb.py from 

```
    messages_to_process = callGAPIpages(service=gmail.users().messages(),
      function='list', items='messages', page_message=page_message,
      userId='me', includeSpamTrash=options.spamtrash, q=options.gmail_search,
      maxItems=500, fields='nextPageToken,messages/id')
```

to

```
    messages_to_process = callGAPIpages(service=gmail.users().messages(),
      function='list', items='messages', page_message=page_message,
      userId='me', includeSpamTrash=options.spamtrash, q=options.gmail_search,
      fields='nextPageToken,messages/id')
```

(That is, just delete `maxItems=500,`)

**Comment by kitgillingham** [comment]
I hit this issue too but using the Windows version. Do you know what I should change in this version to fix it?

Thanks

**Comment by jay0lee** [comment]
Fixed in 1.1

https://git.io/gyb-releases
